pkgs/build-support/dotnet/build-dotnet-module/default.nix

```diff
@@ -138,25 +138,49 @@ in stdenvNoCC.mkDerivation (args // {
       exclusions = dotnet-sdk.passthru.packages { fetchNuGet = attrs: attrs.pname; };
     in writeShellScript "fetch-${pname}-deps" ''
       set -euo pipefail
-      export PATH="${lib.makeBinPath [ coreutils dotnet-sdk nuget-to-nix ]}"
+      export PATH="${lib.makeBinPath [ coreutils dotnet-sdk nuget-to-nix ]}":$PATH
 
+      deps_file="$(realpath "''${1:-/tmp/${pname}-deps.nix}")"
       cd "$(dirname "''${BASH_SOURCE[0]}")"
 
+      while (( $# > 0 )); do
+        case "$1" in
```

Comment:
I wonder if it'd be useful to add an `--output` flag so we dont have any positional flags. Im a bit conflicted myself seeing how you'll basically always want to pass it. Curious what other people think.

Reply:
Not required anymore.